ecommerce/templates/payment/hyperpay.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
```

Comment:
@lgp171188 Is this the client-side checkout implementation? If so, this will also fix our issues with our other client and avoid fetching possibly incomplete profile data from the LMS.